waku/v2/protocol/metadata/pb/generate.go

```diff
@@ -0,0 +1,3 @@
+package pb
+
+//go:generate protoc -I. --go_opt=paths=source_relative --go_opt=Mwaku_metadata.proto=github.com/waku-org/go-waku/waku/v2/protocol/metadata/pb --go_out=. ./waku_metadata.proto
```

Comment:
Would be good to start using proto file from https://github.com/waku-org/waku-proto repo. 
The idea is to not have copies of files and all implementations refer to files from this repo.

Reply:
Indeed. Let's do that as part of https://github.com/waku-org/go-waku/issues/720  
If you wish I can work on it (currently it is assigned to you)

Reply:
Alright.Feel free to go ahead and take it up.